### ISSUE TITLE
Enhance "empty basket"

### ISSUE THREAD

**Issue by yblatti** [easySDI/easySDI#726]
**Original author :** Mérour Xavier

When the basket is empty, it returns a simple "Your basket is empty" to the user... with no other explanations or information.

I think it would be good to have the following behaviour :

- if the basket is empty, the user is redirected to a specific URL (definable in the backend), for example an article.
- if the specific URL is not defined, we keep the current behaviour.

This solution is relatively simple to code, it gives a nice personalization mean... and do not change anything if anyone like the current behaviour ;-)

What do you think ?

**Fixed versions :** 4.3.0

**Comment by yblatti** [comment]
**Original author :** Portier Thomas

Good feature! Ok for me

**Comment by yblatti** [comment]
**Original author :** Mérour Xavier

@Bruno

Do you have any input on this proposal, do you agree ?

**Comment by yblatti** [comment]
**Original author :** Magoni Bruno

+1

**Comment by yblatti** [comment]
**Original author :** Mérour Xavier

Jérôme,

Are you OK with this proposal ?
Or do you have any input ?

Thanks

**Comment by yblatti** [comment]
**Original author :** Teixeira Jérôme

Hello,
Your proposal seems rather consistent and user-oriented. 
OK for this proposal

Thanks

**Comment by yblatti** [comment]
**Original author :** Mérour Xavier

As this feature does not change any "default behaviour" and is a minor enhancement, I think it does not need a formal vote. Right ?

We (Yves) will do this dev. in the coming weeks.

**Comment by yblatti** [comment]
**Original author :** Magoni Bruno

Hi Xavier,

I understand that it could be a minor enhancement but I guess it's important to keep the same process.

I've small questions about URL even if I have already vote on it ;-)
Should it not be always selection of a Joomla article or do you think to other use case ?
I guess it will be more convenient to select a Joomla article than typing some URL ?

Thanks for your feedback...
Bruno

**Comment by yblatti** [comment]
**Original author :** Mérour Xavier

OK for the vote, that's no problem.

```
Should it not be always selection of a Joomla article or do you think to other use case ?
I guess it will be more convenient to select a Joomla article than typing some URL ?
```

Yes, it will be more convenient... BUT they are drawbacks :
- if an administrator would like to point to another resource than "article", for example a special component, it would not be possible
- if you want to specify an item ID (for menu and SEO), it's not automatic. By having a simple URL, the administrator can add parameters he wants in the URL.

To sum up, it's maybe less user-friendly (but it's for the admin !) but more flexible.

**Comment by yblatti** [comment]
**Original author :** Magoni Bruno

OK, thanks for these explanations that I agree with.

**Comment by yblatti** [comment]
**Original author :** Mérour Xavier

Assigned to you... to do later.

**Comment by yblatti** [comment]
**Original author :** Blatti Yves

In my sandbox
